modules/mpv-player/android/src/main/java/expo/modules/mpvplayer/PiPController.kt

```
@@ -106,15 +111,37 @@ class PiPController(private val context: Context, private val appContext: AppCon
+        // setActions(), which would re-register the broadcast receiver
+        // (just unregistered above) and attach play/pause/skip actions to
+        // params being torn down. That leaves a live receiver + stale
+        // actions after the player has unmounted.
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+            try {
+                activity.setPictureInPictureParams(
+                    PictureInPictureParams.Builder()
+                        .setAutoEnterEnabled(false)
+                        .build()
+                )
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)